spectrum/components/drawable.lua

```diff
@@ -9,13 +9,18 @@ local Drawable = prism.Component:extend "Drawable"
 --- Index needs to be a string associated with a sprite in the SpriteAtlas, or
 --- an integer index associated with a sprite.
 --- @param index string|integer
+--- @param size integer?
```

Comment:
Parameter out of order, also missing layer eh. Also need to add size to the overload.